c/misra/test/rules/RULE-17-5/test.c

```diff
@@ -0,0 +1,66 @@
+void f1(int ar[3]);
+void f2(int a, int ar[3]);
+void f3(int *ar);
+void f4(int a, int *ar);
+
+void t1() {
+  int *ar;
+
+  int ar2[3] = {1, 2};
+  int *ar2p = ar2;
+
+  int ar3[3] = {1, 2, 3};
+  int *ar3p = ar3;
+
+  int ar4[4] = {1, 2, 3};
+  int *ar4p = ar4;
+
+  f1(0);    // NON_COMPLAINT
```

Comment:
I don't know how I did this but I botched the spelling of NON_COMPLIANT / COMPLIANT in EVERY test file. Could you please make sure it gets fixed?